communications/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+class EmailLog(models.Model):
+    email_type = models.CharField(max_length=255)
```

Comment:
always avoid unnecessary prefixes,
as model name already has email in it then no need to mention it again

Reply:
model fields should be in alphabetic order
and should be divided in at least 2 section
first fields then related fields aka keys